homeassistant/components/braviatv/config_flow.py

```diff
@@ -91,7 +99,9 @@ async def async_step_user(self, user_input=None):
             errors=errors,
         )
 
-    async def async_step_authorize(self, user_input=None):
+    async def async_step_authorize(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
         """Get PIN from the Bravia TV device."""
         errors = {}
```

Comment:
One more empty collection here.
```suggestion
        errors: dict[str, str] = {}
```